uni-contract-deploy/deploy.yml

```diff
@@ -7,21 +7,12 @@
     - config.yml
   tasks:
   ###############################################################################################################
-    - name: Copy the axon file
-      copy:
-        src: ./templates/deploy.js
-        dest: ./deploy.js
-        mode: 0755
-    - name: Replace url
```

Comment:
There is a replace endpoint operation, which can be replaced in the same way as before, or can be configured to read in deploy.js